docs/Hooks.md

```diff
@@ -82,7 +82,8 @@ fastify.addHook('onResponse', async (res) => {
 | reply | Fastify [Reply](https://github.com/fastify/fastify/blob/master/docs/Reply.md) interface |
 | next | Function to continue with the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) |
 
-It is pretty easy to understand where each hook is executed, if you need a visual feedback take a look to the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) page.
+It is pretty easy to understand where each hook is educated by looking at the [lifecycle page](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md).  
```

Comment:
Hooks don't receive an education 😆

Totally my fault in my last comment. I have no idea where that word came from. I should still be "executed".